### DISCUSSION TITLE
Save a file when new best value is found

### DISCUSSION THREAD

**Discussion by askppp** [optuna/optuna#4655] [Q&A] [answered]
Hello,

how can i save a file (.csv) when everytime a new best value is found ?

I found only that i can access the best_value, but can someone please tell me how to adress that only when best trial is found i can code something ?

What is the appropriate way to code that ? A simple example will help.

Thank you.

**Comment by nzw0301** [comment, marked as answer]
How about this https://optuna.readthedocs.io/en/stable/faq.html#how-can-i-output-a-log-only-when-the-best-value-is-updated?

**Reply by askppp** [reply]
Thank you very much.
